src/main/java/com/code_intelligence/jazzer/autofuzz/FuzzTarget.java

```diff
@@ -210,13 +208,28 @@ public static void fuzzerInitialize(String[] args) {
               throw new Error("Not reached");
             })
             .collect(Collectors.toList());
+
     throwsDeclarations =
         Arrays.stream(targetExecutables)
             .collect(Collectors.toMap(method
                 -> method,
                 method
                 -> Stream.concat(Arrays.stream(method.getExceptionTypes()), alwaysIgnore.stream())
                        .toArray(Class[] ::new)));
+
+    fuzzerInitialize(methodReference, new Meta(targetClass), targetExecutables, null);
+  }
+
+  public static void fuzzerInitialize(
```

Comment:
Since `fuzzerInitialize` has a very specific signature, could we call this something else instead, e.g. just `initialize` or `setTarget`?

We could also add a comment explaining that this is currently only called from JUnit (and internally in this class).

Reply:
I like `setTarget` changed.